mods/cnc/rules/defaults.yaml

```diff
@@ -651,6 +652,7 @@
 	ScriptTriggers:
 
 ^Crate:
+	HiddenUnderShroud:
```

Comment:
Now that we finally have the chance to fix this, let's make it HiddenUnderFog?
Same for other mods of course.